docs/sources/operators-guide/configure/configure-tsdb-block-upload.md

```diff
@@ -7,15 +7,12 @@ weight: 120
 
 # Configure TSDB block upload
 
-Grafana Mimir supports uploading of historic TSDB blocks, sourced from f.ex. Prometheus or
```

Comment:
I wasn't familiar with `f.ex.` as an abbreviation. I figure its the same as `e.g.`? But haven't seen it used as much as e.g. so got rid of it.

Reply:
It's the same as e.g., yes.